editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2056,6 +2056,31 @@ export const MetadataUtils = {
       isJSXConditionalExpression(element.element.value)
     )
   },
+  isRenderPropsFromMetadata(
+    componentMetadata: ElementInstanceMetadataMap,
+    elementPath: ElementPath,
+  ): boolean {
+    const staticElementPath = EP.dynamicPathToStaticPath(elementPath)
+    const selectedElementUid = EP.toUid(staticElementPath)
+    const parentPath = EP.parentPath(elementPath)
+    const element = MetadataUtils.findElementByElementPath(componentMetadata, parentPath)
+    if (
+      element == null ||
+      !isRight(element.element) ||
+      element.element.value.type !== 'JSX_ELEMENT'
+    ) {
+      return false
+    }
+
+    const elementInProps = element.element.value.props.some(
```

Comment:
this maybe the first actual real life case when we are disadvantaged by the element path not being able to express a render prop relationship